src/routes/results.js

```diff
@@ -1,3 +1,79 @@
+const { mockCollection } = require('../../database/schema.js');
+
 module.exports = (req, res) => {
-  res.render('search_results');
+  let options = {};
+  const filterArray = [
+    req.query.search ? 'search-' : 'null-',
+    req.query.stage ? 'stage-' : 'null-',
+    req.query.sector ? 'sector' : 'null'
+  ].join('')
+
+  const findAll = options => {
```

Comment:
this function will probably be used in the landing page for the search element, so I suggest that we move her to a separate file